### ISSUE TITLE
Destructuring problem with const

### ISSUE THREAD

**Issue by rahbari** [google/closure-compiler#3544]
Compiling this code with --language_out ECMASCRIPT_2015
```
function f(p) {
	const {'AAA': a, 'BBB': b} = p;
	console.log(a, b);
}
f({});
```
`java -jar cc20191111.jar --compilation_level ADVANCED --language_out ECMASCRIPT_2015 --js in.js  --js_output_file out.js
`

Result:
`'use strict';var a={};console.log(a.a,a.b);`

Without --language_out:
`var a={};console.log(a.AAA,a.BBB);`

Using let instead of const with --language_out ECMASCRIPT_2015:
`'use strict';{let {AAA:a,BBB:b}={};console.log(a,b)};`

**When language_out is ECMASCRIPT_2015 and const is used for destructuring CC rename fields although it should not.**

**Comment by nreid260** [comment]
I'm unable to reproduce this issue using the most recent version of the compiler. Is this issue fixed by updating versions?

**Comment by rconnamacher** [comment]
I'm encountering this same issue in the latest Compiler binary available on npm (`20200112.0.0`)

**Comment by mprobst** [comment]
I think @brad4d fixed a similar issue recently.

**Comment by rahbari** [comment]
@nreid260 In file root (not function scope) I think it worked correctly.

**Comment by rahbari** [comment]
This is resolved in 20200204
